java/plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/htmlstepdefs/ru/StepDefs.java

```diff
@@ -0,0 +1,66 @@
+package ru.sbtqa.tag.pagefactory.htmlstepdefs.ru;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.ru.И;
+import org.openqa.selenium.NoSuchElementException;
+import ru.sbtqa.tag.pagefactory.exceptions.PageException;
+import ru.sbtqa.tag.pagefactory.exceptions.PageInitializationException;
+import ru.sbtqa.tag.pagefactory.htmlstepdefs.HTMLStepDefs;
+
+public class StepDefs extends HTMLStepDefs {
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @И("^(?:пользователь |он |)в блоке \"([^\"]*)\" \\((.*?)\\)$")
+    public void userActionInBlockNoParams(String block, String action) throws PageInitializationException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)